README_cmake_notes.md

```diff
@@ -0,0 +1,27 @@
+cmake notes:

```

Comment:
What is this file for?

Reply:
nothing. the readme was only for interim reference. removed